DEVELOPER_WORKFLOW.md

```diff
@@ -0,0 +1,191 @@
+# MRE SDK Workflow and Branching Guide
+
+## Motivation for branch system revision
+* minimize the number of custom branch names and purposes, to 3 branches for all non-hotfix commits (2 in SDK, 1 in Unity)
+* remove unnecessary overhead caused by matching AltspaceVR branch changes and releases
+* create a consistent hotfix strategy
+
+## Branches
+* */red and */green = active development. 
+* unity/r0.[minor].n, SDK/r0.[minor].n = release branches – new branches are created when a new release is initiated (not just when we are ready to publish the DLLs/NPM packages)
+* unity/master and sdk/master = branch pointers to minimum supported DLL (typically AltspaceVR's shipped version) and recommended SDK. This is not used for MRE development, and nothing is ever committed to the master branches directly.
+
+Unity, SDK, and Sample repos all have matching branch names (except there is no unity/green – non-breaking features go into the red branch). Samples repo changes are integrated similar to the SDK repo changes. 
+
+<img width='936' height='827' src='branding/Branches.png'/>
+
+## Recommended branch usage
+* use unity/master+sdk/master (or sdk/green) for quick tests/debugging released MREs.
+* use unity/master(or altspacevr retail build)+sdk/green for minor node fixes/hotfixes
+* use unity/red+sdk/red for general development.
+
+## Misc. Notes
+
+(Once we guarantee back compatibility, Unity/master should point to the newest releases sdk, and sdk/master should point to minimum supported version - we want implementers to use the freshest dll, but MRE creators to make MREs that run on every host app). But due to back compat, for now unity/master will point to a version compatible with sdk/master
+
+Minimum supported version=the major.minor version that all host apps support, and have force updated their users to.
+
+Minor version number doesn't *have* to increment in lockstep with altspace's release cycle. We may do more than 1 sdk release between altspace releases.
+
+With this structure, I'd expect there to mostly be 1 or 2 active release versions, but occasionally there would be 3 (or in theory more). The diagram cover 3 active versions for comprehension
+
+(* indicates a detailed workflow below)
```

Comment:
It might be helpful to link to the detailed workflow's anchor from the * location.